internal/plugincommon/consensus/consensus.go

```diff
@@ -15,30 +15,30 @@ import (
 func GetConsensusMap[K comparable, T any](
 	lggr logger.Logger,
 	objectName string,
-	itemsByChain map[K][]T,
+	itemsByKey map[K][]T,
```

Comment:
Thank you, I had the same change pending. But I'm starting to question whether key should be generic since the thresholds become more indirect

Reply:
Good point, lets consider in a followup.